spec/requests/api/v2/posts_request_spec.rb

```diff
@@ -55,29 +55,29 @@
   end
 
   describe 'create a user post', VCR.turn_off! do
-    before :each do 
+    before :each do
       json_response = File.read('spec/fixtures/tone_analysis.json')
-      stub_request(:get, "https://twinword-emotion-analysis-v1.p.rapidapi.com/analyze/?text=Today%20I%20got%20a%20job%20offer%20for%20the%20company%20that%20I%20really%20loved,%20and%20have%20admired%20for%20a%20long%20time.").
-         with(
-            headers: {
-              'Accept'=>'*/*',
-              'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
-              'User-Agent'=>'Faraday v2.6.0',
-              'X-Rapidapi-Host'=>'twinword-emotion-analysis-v1.p.rapidapi.com',
-              'X-Rapidapi-Key'=> ENV['tone_api_key']
-              }).
-         to_return(status: 200, body: json_response, headers: {})
+      stub_request(:get, 'https://twinword-emotion-analysis-v1.p.rapidapi.com/analyze/?text=Today%20I%20got%20a%20job%20offer%20for%20the%20company%20that%20I%20really%20loved,%20and%20have%20admired%20for%20a%20long%20time.')
```

Comment:
Do we want to use VCR here in the future instead of webmock stubbing? It would certainly make it a shorter line...

Reply:
Because this is the API we have to be extra careful with and VCR sometimes needs its cassettes deleted and re-run I wanted to use web mock for this API so that we have more control over it not actually running the API unless we want it to (eg: in postman or production only) - That is why I purposefully stopped VCR and did web mock in all the areas were this is the API that would be called.